### ISSUE TITLE
LaTeX2e 2023-11-01: `\verb*` and `verbatim` print tabs visible in addition

### ISSUE THREAD

**Issue by muzimuzhi** [gpoore/fvextra#19]
For a long time, compared to their un-starred versions, `\verb*` command and `verbatim*` environment only print spaces visible. But since the recently released LaTeX2e 2022-11-01, they print both spaces and tabs visible (see https://github.com/latex3/latex2e/pull/1136), which makes the behavior in LaTeX format and in `fvextra` package different.

`fvextra` package may need to either document the difference or update its behavior.

I've mailed the same info to Herbert Voß, the current maintainer of `fancyvrb`.

```tex
\documentclass{article}
\usepackage{fvextra}
%\usepackage{fancyvrb}

\parindent=0pt
\begin{document}
\subsection*{\Verb|\verb| from the format}
\verb|xtab	tab,sp sp|

\verb*|tab	tab,sp sp|

\subsection*{\Verb|\Verb| from \texttt{fvextra}}
\Verb|xtab	tab,sp sp|

\Verb*|tab	tab,sp sp|

\Verb*[showtabs]|tab	tab,sp sp| % perhaps the new behavior for starred version
\end{document}
```

![image](https://github.com/gpoore/fvextra/assets/6376638/539a2754-a0e4-4982-95b4-c774a68350af)

**Comment by muzimuzhi** [comment]
> I've mailed the same info to Herbert Voß, the current maintainer of `fancyvrb`.

Herbert wrote

> thanks for the report. I'll not change the behaviour of fancyvrb,
> but will write a short note into the documentation.

**Comment by muzimuzhi** [comment]
`fancyvrb` decided to keep its previous behavior.

Starting from v4.5b (2023-11-06), its documentation has included a note (at the end of section "Spaces and tab characters") to point out this difference, see corresponding [diff](https://tug.org/svn/texlive/trunk/Master/texmf-dist/doc/latex/fancyvrb/fancyvrb-doc.tex?pathrev=68772&r1=68772&r2=68771#content) in texlive's svn repo.

**Comment by gpoore** [comment]
I've modified `\Verb*` so that it is equivalent to `showspaces` and `showtabs`. This makes it equivalent to `\verb*`, except without the tab-to-space conversion. I've updated the documentation. Thanks for letting me know about the `\verb*` changes!

At some point, it may be worth adding a `tabstospaces` or similar option that would convert all tabs into spaces. This would allow `fvextra`'s `\Verb*` to be configured exactly as `\verb*`. That might require reworking some of the space-dependent line-breaking algorithms, however.
